src/cloud/docker/docker-manage-cron-jobs.md

```diff
@@ -19,6 +19,14 @@ The Magento cron implementation has the following limitations:
 -  The `setup:cron:run` and `cron:update` commands are not available on Cloud and Docker for Cloud environments
 -  In the Docker environment, cron works only with the CLI container to run the `./bin/magento cron:run` command
 
+By default, cron container is not present in your Docker development environment to improve overall performance.
+
+## Add the cron container
```

Comment:
Level two heading is huge. You can remove it and just rely on the sentence I suggested above, or you can move this heading to appear before the paragraph you added above.

Reply:
Fixed